examples/render_to_image_widget.rs

```diff
@@ -150,12 +151,13 @@ fn render_to_image_example_system(
     preview_cube_query: Query<&Handle<StandardMaterial>, With<PreviewPassCube>>,
     main_cube_query: Query<&Handle<StandardMaterial>, With<MainPassCube>>,
     mut materials: ResMut<Assets<StandardMaterial>>,
+    windows: Query<Entity, With<Window>>,
 ) {
     let cube_preview_texture_id = egui_ctx.image_id(&cube_preview_image).unwrap();
     let preview_material_handle = preview_cube_query.single();
     let preview_material = materials.get_mut(preview_material_handle).unwrap();
 
-    let ctx = egui_ctx.ctx_mut();
+    let ctx = egui_ctx.ctx_for_window_mut(windows.iter().next().unwrap());
```

Comment:
that's a loss of ergonomics indeed, I look forward to what can be done component-wise :) ; egui_context stored in a component attached to a window is my first not-much-informed thought

Reply:
Now this looks like:
```rs
egui_ctx: Query<&EguiContext, With<Window>>,
[...]
let ctx = egui_ctx.iter().next().unwrap();
```
Thoughts?
The `With<Window>` isn't really needed, but maybe clarifies things. idk if it makes sense or not.